docs/layers/js-tutorial.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Tutorial
+sidebar_label: Example in Javascript
+slug: /layers/js-tutorial
+---
+
+# Using a Layer in Javascript
+
+This example shows how to setup a Layer with multiple parameters and then run multiple experiments with different combinations of those parameters.
+
+## Create a Layer
```

Comment:
This whole guidance seems really good, but all belongs under a "Setup" category, likely